### DISCUSSION TITLE
Dependency Grammar literature

### DISCUSSION THREAD

**Discussion by Fourthought** [explosion/spaCy#7868] [Help: Other Questions] [answered]
Hi there,

Hopefully a quick question. I'm looking to document a dependency parsing component and note spaCy uses dependency grammar, as opposed to Chomsky's phrase structure grammar.

Do you have any go to books, papers or references about dependency grammar please.

Steve

**Comment by polm** [comment, marked as answer]
spaCy's annotation scheme is based on [Universal Dependencies](https://universaldependencies.org/). Besides their official docs I would recommend Chapter 14 in the [Jurafsky & Martin NLP Book](https://web.stanford.edu/~jurafsky/slp3/), which is probably a more organized and accessible introduction.
